ElastosDIDSDK/DID.swift

```diff
@@ -198,10 +261,11 @@ extension DID: Equatable {
     }
 }
 
-extension DID: Hashable {
-    public func hash(into hasher: inout Hasher) {
-        hasher.combine(self.toString())
-    }
+extension DID {
```

Comment:
Be sure to remove the extension from Equatable. It could be wrong if DID is as a Key of map object.